docs/features/virtual_folders.rst

```diff
@@ -0,0 +1,69 @@
+.. _virtual_folders:
+
+Virtual Folders
+===============
+
+.. versionadded:: 2.7
+
+Virtual folders provide a way to group translations based on any criteria,
+including a same file across all the languages in a project, or specific files.
+Virtual folders have priority, so they can be used to allow translators to
+focus on the most important work.
+
+
+.. _virtual_folders#attributes:
+
+Virtual folders attributes
+-----------------------------------
+
+Virtual folders have several attributes:
+
+- A mandatory lowercase name,
+- A mandatory location,
+- An optional priority,
+- An optional browsability flag,
+- An optional description,
+- An optional filtering rule.
+
+Each virtual folder must have a different name and location pair. This means
```

Comment:
Do you mean that name + location must be unique? or that name and location must differ?